MANIFEST.in

```diff
@@ -1,7 +1,48 @@
-include LICENSE
 
-# added by check-manifest
+   
+include LICENSE
+include CONTRIBUTING.rst
+include README.md
+include CHANGELOG.md
+include package.json
+include bower.json
+include .bowerrc
+include pyproject.toml
+include setup.py
+include setupbase.py
+include Dockerfile
+include *.js
+include *.in
 include *.md
-recursive-include docs *.md
-recursive-include nbclassic *.py
+include *.svg
+include *.yml
+recursive-include nbclassic *.less
 recursive-include tests *.py
+graft jupyter_server_config.d
+graft tools
+graft nbclassic/tests
+
+exclude git-hooks
```

Comment:
These should be `prune`: https://packaging.python.org/en/latest/guides/using-manifest-in/#manifest-in-commands